Makefile

```
@@ -348,11 +348,16 @@ sast-iac: ## Run IaC security scan (trivy)
 sast-secret: ## Run gitleaks secret scan
```

checkmake (0.3.2)

[warning] Target body for "sast-secret" exceeds allowed length of 5 lines (12).

(maxbodylength)